README.md

```diff
@@ -175,3 +175,37 @@ split:
     - git fetch --prune --unshallow || git fetch --prune
     - gitsplit --ref "${CI_COMMIT_REF_NAME}"
 ```
+
+# Sample with Github Actions
+
+I recommend to not use [the checkout action](https://github.com/actions/checkout) because it will checkout only the last
```

Comment:
What's about https://github.com/actions/checkout#fetch-all-history-for-all-tags-and-branches

```yaml
- uses: actions/checkout@v2
  with:
    fetch-depth: 0
```

Reply:
I did try a lot of combinations when I tried to use checkout action and never had something that had what I needed, if you prefer I can update the README with this syntax but I can't say if it's working or not.